core/src/main/java/com/klaviyo/coresdk/networking/NetworkBatcher.kt

```diff
@@ -37,9 +37,6 @@ object NetworkBatcher {
         }
 
         for (request in requests) {
-            if (request is TrackRequest) {
```

Comment:
Unrelated to the rest of the PR, but just noticed that this code here was totally not necessary and this timestamp generation was moved to an earlier point in a previous PR.